frontend/app/services/lookup-service.server.ts

```diff
@@ -178,6 +189,47 @@ function createLookupService() {
     throw new Error(`Failed to fetch data. Status: ${response.status}, Status Text: ${response.statusText}`);
   }
 
+  async function getAllFirstNationsTypes() {
+    const url = `${INTEROP_API_BASE_URI}/lookups/first-nations-types/`;
+    const response = await fetch(url);
+
+    const firstNationsTypeSchemaList = z.array(firstNationsTypeSchema);
+
+    if (response.ok) {
+      return firstNationsTypeSchemaList.parse(await response.json());
+    }
+
+    log.error('%j', {
+      message: 'Failed to fetch data',
+      status: response.status,
+      statusText: response.statusText,
+      url: url,
+      responseBody: await response.text(),
+    });
+
+    throw new Error(`Failed to fetch data. Status: ${response.status}, Status Text: ${response.statusText}`);
+  }
+  async function getAllFirstNationsEnumTypes() {
+    const url = `${INTEROP_API_BASE_URI}/lookups/first-nations-enum-types/`;
+    const response = await fetch(url);
+
+    const getAllFirstNationsEnumTypeSchemaList = z.array(getAllFirstNationsEnumTypeSchema);
```

Comment:
```suggestion
    const firstNationsEnumTypeSchemaList = z.array(firstNationsEnumTypeSchema);
```

Reply:
please if its ok now, ive changed the name and all